Library/Formula/hue.rb

```diff
@@ -0,0 +1,157 @@
+class Hue < Formula
+  desc "Hue is a Web interface for analyzing data with Apache Hadoop."
+  homepage "http://gethue.com/"
+  url "https://github.com/cloudera/hue/archive/release-3.8.1.tar.gz"
+  # version "3.8.1" # redundant with version scanned from URL
+  sha256 "582777f567b9f4a34e4ce58cfd5ed24aff15f3e02f193e7990329d8cb0161a8a"
+
+  # system dependencies - either Homebrew or Mac OS versions
+  depends_on :java
+  depends_on :python
+  depends_on :mysql
+
+  depends_on "maven" => :build
+  depends_on "openssl" => :optional
```

Comment:
This needs to be mandatory or you need to find a way to tell Hue's build scripts not to build against openssl.